ROADMAP.js-ipfs.md

```diff
@@ -0,0 +1,40 @@
+js-ipfs roadmap
+===============
+
+> Roadmap for IPFS implementation in JavaScript. This repo describes the milestones and each of the steps that must be accomplished in order to get js-ipfs ready for people to use
+
+note: libp2p, which is the network stack of IPFS, has its own ROADMAP and should be referenced as `libp2p`, a component, in this project.
+
+## Milestones
+
+#### v0.1.0
+
+> main goal: Working version with a js-ipfs delegated node
+
+- [ ] writable gateway
```

Comment:
when i list the commands, i dont mean using the api, i just mean "commands implemented to operate on a node, like, implementing this:

```
ipfs.object.get = function(node, key, cb) { }
// or
ipfs.object.get = function(key, cb) { } // if `this` is the node.
```